src/vulkan/vk-api.cpp

```
@@ -112,6 +132,29 @@ Result VulkanApi::initGlobalProcs(const VulkanModule& module)
     return SLANG_OK;
 }
 
+Result VulkanApi::initEnumerationProcs(VkInstance instance)
+{
+    SLANG_RHI_ASSERT(instance && vkGetInstanceProcAddr != nullptr);
+
+    vkEnumeratePhysicalDevices =
+        (PFN_vkEnumeratePhysicalDevices)vkGetInstanceProcAddr(instance, "vkEnumeratePhysicalDevices");
+    vkGetPhysicalDeviceProperties2 =
+        (PFN_vkGetPhysicalDeviceProperties2)vkGetInstanceProcAddr(instance, "vkGetPhysicalDeviceProperties2");
+    vkEnumerateDeviceExtensionProperties =
```

Cppcheck (2.20.0)

[style] The function 'calcAlignedDown' is never used.

(unusedFunction)